server/routerlicious/packages/lambdas/src/foreman/lambda.ts

```diff
@@ -3,23 +3,11 @@
  * Licensed under the MIT License.
  */
```

Comment:
it feels like we should kill whole lamba, no?

Reply:
Yes! And I got confirmation from the server teams to do it as well!